tests/serac_dtor.cpp

```diff
@@ -24,6 +28,22 @@ int main(int argc, char **argv)
 
   MPI_Barrier(MPI_COMM_WORLD);
 
+  const char *mesh_file = "NO_MESH_GIVEN";
+  // Parse command line options
+  mfem::OptionsParser args(argc, argv);
+  args.AddOption(&mesh_file, "-m", "--mesh", "Mesh file to use.", true);
+  args.Parse();
+  if (!args.Good()) {
+    if (myid == 0) {
+      args.PrintUsage(std::cout);
+    }
+    MPI_Finalize();
+    return 1;
+  }
+  if (myid == 0) {
+    args.PrintOptions(std::cout);
+  }
+
   std::ifstream imesh("../../data/beam-hex.mesh");
```

Comment:
`std::ifstream imesh(mesh_file)`

`std::ifstream` already has a way to check if the file exists, maybe we can move away from the platform-specific implementation of `file_exists`